_ingest-pipelines/processors/kv.md

```diff
@@ -0,0 +1,147 @@
+---
+layout: default
+title: KV
+parent: Ingest processors
+nav_order: 200
+redirect_from:
+   - /api-reference/ingest-apis/processors/lowercase/
+---
+
+# KV processor
+
+The `kv` processor automatically extracts specific event fields or messages that are in a`key=value` format. This structured format organizes your data by grouping it together based on keys and values. It's helpful for analyzing, visualizing, and using data such as user behavior analytics, performance optimizations, or security investigations. 
+
+## Example
+The following is the syntax for the `kv` processor: 
+
+```json
+{
+  "kv": {
+    "field": "message",
+    "field_split": " ",
+    "value_split": " "
+  }
+}
+```
+{% include copy-curl.html %}
+
+## Configuration parameters
+
+The following table lists the required and optional parameters for the `kv` processor.
+
+| Parameter  | Required/Optional  | Description  |
+|---|---|---|
+`field`  | Required  | The name of the field that contains the data to be parsed. Supports [template snippets]({{site.url}}{{site.baseurl}}/ingest-pipelines/create-ingest/#template-snippets). |
+`field_split` | Required | The regex pattern for key-value pair splitting. |
+`value_split` | Required | The regex pattern for splitting the key from the value within a key-value pair, for example, equal sign `=` or colon `:`.
+`exclude_keys` | Optional | The keys to exclude from the document. Default is `null`. |
+`include_keys` | Optional | The keys for filtering and inserting. Default is to include all keys. |
+`prefix` | Optional | The prefix to add to the extracted keys. Default is `null`. |
+`strip_brackets` | Optional | If set to `true`, strips brackets `()`, `<>,` or `[]` and quotes `'` or `"` from extracted values. Default is `false`.
```

Comment:
Can we put parentheses around the examples?: (`()`, `<>,` or `[]`) and quotes (`'` or `"`)